overlord/snapstate/autorefresh.go

```diff
@@ -398,6 +402,10 @@ func (m *autoRefresh) launchAutoRefresh() error {
 	chg.Set("snap-names", updated)
 	chg.Set("api-data", map[string]interface{}{"snap-names": updated})
 
+	tm.Stop()
```

Comment:
Could we put this into a `defer func() {...}` maybe to have it close to the timings.New() call?